receiver/googlecloudspannerreceiver/config.go

```diff
@@ -0,0 +1,114 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googlecloudspannerreceiver
+
+import (
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+)
+
+type Config struct {
+	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
+
+	TopMetricsQueryMaxRows int       `mapstructure:"top_metrics_query_max_rows"`
```

Comment:
How about using a struct `TopMetricsQuery` with a field inside `max_rows` to allow more options to be added later

Reply:
Nice idea. For now we do not expect more options.
We can introduce such structure later if required(more options appear).